app/controllers/organizations/staff/default_pet_tasks_controller.rb

```diff
@@ -1,11 +1,20 @@
 class Organizations::Staff::DefaultPetTasksController < Organizations::BaseController
   before_action :context_authorize!, only: %i[index new create]
   before_action :set_task, only: %i[edit update destroy]
+  include ::Pagy::Backend
 
   layout "dashboard"
 
   def index
-    @default_pet_tasks = authorized_scope(DefaultPetTask.all).sort_by { |task| task.species }
+    ensure_due_in_days_in_q_params
```

Comment:
What is the `ensure_due_in_days_in_q_params` method for and why do we do this only for this parameter?

If we must have it, can we move `ensure_due_in_days_in_q_params` this to a before_action filter for index?